appendix/google-gke.md

```diff
@@ -0,0 +1,54 @@
+---
+label: Starting Google GKE Cluster
+icon: /images/gke.png
+order: 890
+---
+
+# Launching an GKE cluster
+Below we describe how we can launch and use Kontain in an Amazon EKS (Kubernetes) cluster.
+
+## Starting up an GKE cluster
+
+### Pre-requisites
+Take the following steps to enable the Kubernetes Engine API:
+- [Sign in to your Google Cloud account](https://console.cloud.google.com/freetrial?_ga=2.49271654.1392909812.1664219973-2131462911.1664219973). If you're new to Google Cloud, [create an account](https://console.cloud.google.com/freetrial?_ga=2.49271654.1392909812.1664219973-2131462911.1664219973) to evaluate how our products perform in real-world scenarios. 
+
+- In the Google Cloud console, on the project selector page, select or [create a Google Cloud project](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+
+!!!
+Note: If you don't plan to keep the resources that you create in this procedure, create a project instead of selecting an existing project. After you finish these steps, you can delete the project, removing all resources associated with the project.
+!!!
+
+- [Go to project selector](https://console.cloud.google.com/projectselector2/home/dashboard)
+
+- Make sure that billing is enabled for your Cloud project. Learn how to [check if billing is enabled on a project](https://cloud.google.com/billing/docs/how-to/verify-billing-enabled).
+- [Enable the Artifact Registry and Google Kubernetes Engine APIs](https://console.cloud.google.com/flows/enableapi?apiid=artifactregistry.googleapis.com,container.googleapis.com).
+
+- [Install Kubectl auth plugin](https://cloud.google.com/blog/products/containers-kubernetes/kubectl-auth-changes-in-gke)
+
+### Create cluster using Kontain helper script
+
+Download helper script and make sure it is executable
+
+```shell
+curl -o gce-cluster.sh https://raw.githubusercontent.com/kontainapp/k8s-deploy/master/helpers/eks-cluster.sh
```

Comment:
the curl is pointing to eks-cluster.sh - think it should be downloading gce-cluster.sh

Reply:
Also think it probably should be gke-cluster.sh since cluster name is GKE.